### PR TITLE
bump version 2.3.0dev

### PR DESCRIPTION
**Pull request by Daniel-VM**
## PR checklist

- [X] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/bacass/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/bacass _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [X] Make sure your code lints (`nf-core lint`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [X] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

## PR description

Bump version 2.3.0dev

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,18 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v2.3.0dev nf-core/bacass: "Navy Iron Oyster" 2024/03/17
```
Comment (d4straub) [inline CHANGELOG.md:6]: probably remove the date...

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,18 @@
...
+
+### `Added`
+
+### `Fixed`
+
+### `Dependencies`
+
+### `Deprecated`
+
 ## v2.2.0 nf-core/bacass: "Aqua Platinum Zebrafish" 2024/03/27
```
Comment (d4straub) [inline CHANGELOG.md:18]: the date here seems wrong?

**Review by d4straub — APPROVED** [review]
LGTM, just 2 comments

**Comment by d4straub** [comment]
There are a few warning that could be fixed: 
```
WARN: A process with name 'MINIMAP2_CONSENSUS' is defined more than once in module script: /home/runner/work/bacass/bacass/./workflows/bacass.nf -- Make sure to not define the same function as process
WARN: A process with name 'MINIMAP2_POLISH' is defined more than once in module script: /home/runner/work/bacass/bacass/./workflows/bacass.nf -- Make sure to not define the same function as process
```

File: CHANGELOG.md (hunk 2 above)
Comment (d4straub) [reply CHANGELOG.md:18]: I usually put there the date of the release, not the date of the PR. But probably that is fine too, didnt think that this is a possibility :D

Comment (Daniel-VM) [reply CHANGELOG.md:18]: got it 😅 . I will proceed with the update ;).

**Comment by Daniel-VM** [comment]
> There are a few warnings that could be fixed in another PR:
> 
> ```
> WARN: A process with name 'MINIMAP2_CONSENSUS' is defined more than once in module script: /home/runner/work/bacass/bacass/./workflows/bacass.nf -- Make sure to not define the same function as process
> WARN: A process with name 'MINIMAP2_POLISH' is defined more than once in module script: /home/runner/work/bacass/bacass/./workflows/bacass.nf -- Make sure to not define the same function as process
> ```

Those are old friends of mine hahaha. I'm going to put an issue. At the moment I have not been able to reproduce the warnings locally.

**Pull request merged**